README.md

```diff
@@ -1,2 +1,45 @@
 # chpl-api-for-dummies
 Provides an example of an application that calls the CHPL REST Service
+
+# Installation instructions
+
+## Install required software
+
+Java 1.8.0
+Eclipse (latest)
```

Comment:
Is Eclipse actually required? I was hoping someone with just Java would be able to build/run this from the command line